masonite/request.py

```diff
@@ -41,11 +41,13 @@ def __init__(self, environ=None):
         self.encryption_key = False
         self.container = None
 
-    def input(self, name):
+    def input(self, name, default_val=False):
```

Comment:
This whole thing is perfect but it is such a pet peeve of mine to shorten down variable names. Especially when it's only shortened down by 2 characters lol. Please just make this `default` instead.